source/AI.cpp

```diff
@@ -1254,7 +1254,7 @@ void AI::MoveEscort(Ship &ship, Command &command) const
 			const System *from = ship.GetSystem();
 			
 			// Check how much fuel is required to reach the next refuel system.
-			if(systemHasFuel && hasFuelCapacity && ship.Fuel() < 1.)
```

Comment:
`hasFuelCapacity` is not guaranteed to always be true (there is a discussion to remove `!hasFuelCapacity` from AI.cpp:1226, which if done would require checking it here).